.eslintrc

```diff
@@ -158,6 +158,18 @@
     }
   },
   "overrides": [
+    {
+      "files": ["*.jsx", "*.tsx"],
+      "rules": {
+        "no-unconditional-metabase-links-render": "error"
```

Comment:
This rule should only test JSX components because we expect to check if React components render the Metabase links conditionally or not. We don't care about links created outside of React component files.